lib/stimulus_reflex/broadcasters/selector_broadcaster.rb

```diff
@@ -64,5 +58,26 @@ def selector?
     def to_s
       "Selector"
     end
+
+    private
+
+    def create_update_collection(selectors, html)
+      updates = selectors.is_a?(Hash) ? selectors : {selectors => html}
+      updates.map do |key, value|
+        Update.new(key, value, reflex)
+      end
+    end
+
+    StimulusReflex::SelectorBroadcaster::Update = Struct.new(:key, :value, :reflex) do
+      def selector
+        identifiable?(key) ? dom_id(key) : key.to_s
+      end
+
+      def html
+        html = reflex.render(key) if key.is_a?(ActiveRecord::Base) && value.nil?
+        html = reflex.render_collection(key) if key.is_a?(ActiveRecord::Relation) && value.nil?
+        html || value
+      end
+    end.include(CableReady::Identifiable)
```

Comment:
Wow, am I ever not familiar with this syntax. Does this just include it in each instance of the Struct?

Can you not just use `include` inside the block as you would with a class?

Reply:
yep, you can include modules at runtime

and no, can't do this with a `Struct`, and I was reluctant to make it a full class. However, if desired, I can rewrite this

Reply:
I don't think that's necessary?